src/java/org/apache/cassandra/net/SocketFactory.java

```diff
@@ -228,39 +228,41 @@ static SslHandler newSslHandler(Channel channel, SslContext sslContext, @Nullabl
         return sslHandler;
     }
 
-    static String encryptionLogStatement(EncryptionOptions options)
+    /**
+     * Summarizes the intended encryption options, suitable for logging. Once a connection is established, use
+     * {@link SocketFactory#encryptionConnectionSummary} below.
+     * @param options options to summarize
+     * @return description of encryption options
+     */
+    static String encryptionOptionsSummary(EncryptionOptions options)
```

Comment:
Would it be better to simply change this to:
```java
if (options == null)
    return "disabled";

String encryptionType = SSLFactory.openSslIsAvailable() ? "openssl" : "jdk";
return options.tlsEncryptionPolicy().description() + '(' + encryptionType + ')';
```

Reply:
Outputing `unencrypted(openssl)` read badly to me, and I wanted to enforce the same description of encrypted/optional/unencrypted from the enum so I was also getting rid of `disabled`.